test/integration/controller/jobset_controller_test.go

```diff
@@ -158,7 +158,10 @@ var _ = ginkgo.Describe("JobSet validation", func() {
 				{
 					shouldSucceed: true,
 					fn: func(js *jobset.JobSet) {
-						js.Spec.Suspend = pointer.Bool(true)
+						var newjs jobset.JobSet
+						gomega.Eventually(k8sClient.Get(ctx, types.NamespacedName{Name: js.Name, Namespace: js.Namespace}, &newjs), timeout, interval).Should(gomega.Succeed())
```

Comment:
We don't need Eventually, Expect should suffice because the JobSet should exist.

Reply:
done.